### DISCUSSION TITLE
hnn_core package upgrade

### DISCUSSION THREAD

**Discussion by JBruce32** [jonescompneurolab/hnn-core#580] [Q&A]
Hello, I am trying to upgrade the hnn_core package to 0.3, because I would like to use the current way of optimizing parameters. When I try to upgrade the version with pip it is only able to get 0.2. How do I get around this issue?

Best,
Joshua

**Comment by jasmainak** [comment]
Hi Joshua, actually 0.3 has not been released yet. But you may use the development version by doing:

```py
$ pip install --upgrade https://api.github.com/repos/jonescompneurolab/hnn-core/zipball/master
```

let us know if it works
